script.py

```diff
@@ -217,7 +223,20 @@ def parse_commit_content(commit_msg):
 with open(meta_runtime_path, 'w') as f:
     yaml.dump(dataRuntimeMap, f, sort_keys=False, default_flow_style=False, explicit_start=True, Dumper=Dumper)
 
-# 5. Update CHANGELOG.rst & galaxy.yml and push a releasing PR
+# 5. Update annotation unsupported classes
+annotation_unsupported_file = Path('{0}/plugins/module_utils/annotation_unsupported.py'.format(directory))
+apic = os.getenv("APIC")
```

Comment:
Can we default to the devnet one when no environment variable is defined?

Reply:
Yes we can. :)

Reply:
Done 👍